src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  currentTodo: Todo | null,
+  todo: Todo,
+  showTodoUser: (value: Todo) => void,
+};
+
+export const TodoItem: FC<Props> = ({ todo, currentTodo, showTodoUser }) => {
+  const { id, title, completed } = todo;
+
+  return (
+    <>
+      <td className="is-vcentered">{id}</td>
+      <td className="is-vcentered">
+        <span className="icon" data-cy="iconCompleted">
+          {completed
+            ? (
+              <i className="fas fa-check" />
+            )
+            : ''}
+        </span>
+      </td>
+
+      <td className="is-vcentered is-expanded">
+        <p className={
+          !completed ? 'has-text-danger' : 'has-text-success'
+        }
+        >
```

Comment:
```suggestion
        <p className={
          !completed ? 'has-text-danger' : 'has-text-success'
        }>
```

Reply:
![image](https://github.com/mate-academy/react_redux-list-of-todos/assets/105709942/dd44aad7-5268-4f64-b7c9-f54b6a68b3c6)
the linter swears